src/Gui/DlgProjectInformationImp.cpp

```diff
@@ -29,11 +29,28 @@
 #endif
 
 #include <App/Document.h>
+#include <App/License.h>
 
 #include "DlgProjectInformationImp.h"
 #include "ui_DlgProjectInformation.h"
 
 
+#if 0 // needed for Qt's lupdate utility
+    qApp->translate("Gui::Dialog::DlgSettingsDocument", "All rights reserved");

```

Comment:
Why not do this in 'License::init()' using 'QT_TRANSLATE_NOOP'?

Reply:
Because i18n is not yet supported in App modules. Once it's done the code can move to License.cpp

Reply:
Thanks for insight.